reversing levels of predictor does not flip adjusted difference/95% CI

emilyvertosick opened the pull request:
If the levels of the x variable are switched, the difference/adjusted difference and corresponding 95% CI should be flipped accordingly. This works correctly for unadjusted but did not work for adjusted.

`tbl_propdiff(data = trial, x = "trt", y = "response", formula = "{y} ~ {x} + age + stage", method = "boot_sd", bootstrapn = 25)`

![image](https://user-images.githubusercontent.com/49212917/96776635-486d5600-13b7-11eb-89a3-faca3af1bac6.png)

`tbl_propdiff(data = trial %>% mutate(trt = fct_rev(trt)), x = "trt", y = "response", formula = "{y} ~ {x} + age + stage", method = "boot_sd", bootstrapn = 25)`

![image](https://user-images.githubusercontent.com/49212917/96776352-f0365400-13b6-11eb-9ce7-014621ddfebd.png)

The columns are switched but the adjusted difference/95% CI are not.

**What changes are proposed in this pull request?**
Added "arrange" statement to order variable labels to get the correct difference.

**If there is an GitHub issue associated with this pull request, please provide link.**


--------------------------------------------------------------------------------

Checklist for PR reviewer

- [ ] PR branch has pulled the most recent updates from master branch
- [ ] NEWS.md has been updated
- [ ] `usethis::use_spell_check()` runs with no spelling errors in documentation
- [ ] If a new function was added, function included in `pkgdown.yml`
- [ ] If a bug was fixed, a unit test was added for the bug check
- [ ] Run `pkgdown::build_site()`. Check the R console for errors, and review the rendered website.
- [ ] Code coverage is suitable for any new functions/features. Review coverage with `covr::report()`.
- [ ] R CMD Check runs without errors, warnings, and notes

If you **are NOT** making a release

- [ ] When the branch is ready to be merged into master, increment the version number using `usethis::use_version(which = "dev")` and merge the PR.

If you **are** making a release

- [ ] When the branch is ready to be merged into master, increment the version number using `usethis::use_version(which = "patch")` and merge the PR.
- *After the pull request has been merged*
  - Run `usethis::use_github_release()`
  - Publish the GitHub release

ddsjoberg commented:
If you're ok with the images I can merge without review.

But can you add until tests to make sure we don't break it later?

emilyvertosick commented:
> If you're ok with the images I can merge without review.
> 
> But can you add until tests to make sure we don't break it later?

I added a unit test to check that the estimate is flipped correctly for both univariate and multivariable models in tbl_propdiff. I updated the NEWS.md file but let me know if you'd like me to do this as a release
